modules/migration-configuring-proxies.adoc

```diff
@@ -44,6 +44,16 @@ If you scale up workers that are not included in the network defined by the `net
 +
 This field is ignored if neither the `httpProxy` nor the `httpsProxy` field is set.
 
+. To configure supplemental groups to Rsync pods, update the `MigrationController` CR as follows:
```

Comment:
Is this step needed? Did I indicate correctly that the spec secion might contain other items?

Reply:
@RichardHoch this step should not be needed here as you already added configuring supplemental groups as a separate section. 
This section should only contain proxy related stuff.